apps/web/app/page.tsx

```diff
@@ -1,16 +1,24 @@
 // 3rd party
 import { JSX } from "react";
 // repo
-import Link from "@repo/ui/link";
 import { URL_WALKTHROUGH_HREF } from "@repo/constants/src/urls";
-import ModalSide from "@repo/ui/modal-side";
+import WalkthroughCard from "@repo/ui/walkthrough-card";
+import { WalkthroughJSONData } from "../../../packages/data/src/useWalkthroughData";
```

Comment:
Needs to use the `@repo/data..` import

Reply:
Nice I didn't know @repo was configured to support that. This is why I mentioned adding absolute imports! haha